docs/content/commands/npm-adduser.md

```diff
@@ -12,6 +12,8 @@ npm adduser [--registry=url] [--scope=@orgname] [--always-auth] [--auth-type=leg
 aliases: login, add-user
 ```
 
+Note: This command is unaware of workspaces.
```

Comment:
maybe instead of just "unaware", commands like this should be marked to indicate that it wouldn't make sense for them to be workspace-aware?

(altho in this case i can imagine wanting to log in to specific users only in specific workspaces' .npmrc files)

Reply:
> maybe instead of just "unaware", commands like this should be marked to indicate that it wouldn't make sense for them to be workspace-aware?

this is me marking them to indicate that it wouldn't make sense for them to be workspace-aware 😅

Reply:
Most of these do make sense for it, though :-)